share/completions/ip.fish

```diff
@@ -194,6 +194,12 @@ function __fish_ip_scope
         host "Address is only valid on this host"
 end
 
+function __fish_list_netnses
+    ip netns list | while read a b c
+        echo $a
```

Comment:
It looks like it is allowed to create a network namespace with a name that starts with `-`. So let's use `echo --` or `printf`, just in case ;)

Reply:
Fair point. Fixing.